packages/communication-react/package.json

```diff
@@ -39,6 +39,7 @@
     "events": "^3.3.0",
     "html-react-parser": "^5.1.10",
     "immer": "10.0.4",
+    "libphonenumber-js": "^1.11.2",
```

Comment:
Not sure anyone will see this.

This repo is 550kb, compressed.  The author is based in Moscow, Russia and has had his account banned from Github.  His CV includes working for "Russian NSA-like system for intercepting communication on the internet with the purpose of preventing governmental threats".
https://docs.google.com/document/d/1B2C1IGwZjXTxdogOvE5vejcBx0kKe1ynHXzPxQcv6x4/edit

Y'all are far more trusting than I would be for a library that isn't doing any heavy lifting.

Reply:
Thanks for reporting this, we'll take a look and reassess this package usage.

Reply:
#4920

Reply:
Really fast turn-around.  Thank you for the promptness!